search-and-add-github-repos.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+set -ex
+
+# This script is exclusively developed for parser development purposes. It serves the specific use case of ingesting GitHub repositories internally for SAAS.
+
+# Create out folder if not exist
+mkdir -p out
+
+# Search and add github repos by language, change the below language
+
+# generate `gh.json` which includes repo fullName and branch name,
+# Manually customize the language and adjust the count below according to your specific requirements.
+# option to add filter: --updated="> 2023-11-21"
+gh search repos --language javascript --visibility public --limit 1000 --json fullName,defaultBranch > out/gh.json
```

Comment:
Might as well make `javascript` a script parameter. In addition we could also document that it could be interesting to set `--sort stars` (and / or `--updated "> $(date -v-1m "+%Y-%m-%d")"` to get repos updated in the last month), as users might be more inclined to run recipes against popular repos.

Reply:
yeah, I thought about this to put both language and count as script parameters, and then I think it will be used by us internally and we might want to do other customization, so I leave it as hard coded since anyway we need to type the language somewhere. We can update this for any needs for sure.